TestBenches/FileReadUtility.h

```diff
@@ -152,7 +153,6 @@ template<class MemType, int OutputBase=2, int LSB=-1, int MSB=-1>
 unsigned int compareMemWithMem(const MemType& memory_ref, const MemType& memory,
                                    const int ievt, const std::string& label,
                                    const bool truncated = false,
-                                   const bool print_empty = false,
```

Comment:
@aryd Is there a reason for removing this argument? I added it in #333 for the TrackBuilder test bench, because otherwise, if the first track in an event doesn't have a matched stub in a given layer/disk, none of the stub words for that layer/disk are printed for subsequent tracks.

Reply:
Restored this update.